starknet_py/net/account/account_client.py

```diff
@@ -407,10 +484,13 @@ async def create_account(
 async def deploy_account_contract(
     client: Client, public_key: int
 ) -> AddressRepresentation:
-    deploy_tx = make_deploy_tx(
-        constructor_calldata=[public_key],
-        compiled_contract=COMPILED_ACCOUNT_CONTRACT,
-    )
+    with warnings.catch_warnings():
```

Comment:
We should add a deprecation warning for `deploy_account_contract`, stating that it will be removed when deploy tx is removed.